packages/devtools_app/lib/src/screens/memory/memory_analyzer.dart

```diff
@@ -517,7 +527,7 @@ class AnalysisInstanceViewTable extends StatefulWidget {
 /// Table of the fields of an instance (type, name and value).
 class AnalysisInstanceViewState extends State<AnalysisInstanceViewTable>
     with AutoDisposeMixin {
-  MemoryController controller;
+  MemoryController? controller;
```

Comment:
make late. Use the `initialized` pattern we've been using elsewhere

Reply:
done